src/langcheck/metrics/en/_openai.py

```diff
@@ -45,10 +45,7 @@ def get_score(self, prompt: str) -> float:
 
         Returns:
             Score associated with the given prompt based the resulting
-                assessment
-
-        Raises:
-            AssertionError: If the OpenAI API returns an unrecognized assessment
+                assessment, `None` if the score could not be computed
```

Comment:
Would it be helpful to explicitly say that when the score is not computed there are some issues about OpenAI APIs?

Reply:
It could be due to OpenAI not returning a value in the format specified in the prompt as well, which is a little different from an issue with the API. Added some more detail in the function's docstring